src/components/assets/three-dots.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="992px" height="992px" viewBox="0 0 992 992" style="enable-background:new 0 0 992 992;" xml:space="preserve">
+<g>
+	<circle cx="144.3" cy="496" r="144.3"/>
+	<circle cx="496" cy="496" r="144.3"/>
+	<circle cx="847.7" cy="496" r="144.3"/>
+</g>
+<g>
```

Comment:
I believe empty groups for elements are not needed here, you can always use something like https://www.svgminify.com/ to cut all unneeded stuff from svg